<section id="schedule">
  <h3>Wednesday, Nov 13</h3>
  <table class="table table schedule table-striped" role="presentation">
    <tbody>
      <tr><td>BEFORE</td><td><a href="https://smcclatchy.github.io/qtl-mapping/index.html#software-setup">Setup Instructions</a></td>
        <td> Download files required for the lesson</td>
      </tr>
      <tr><td>9:00</td>
        <td>0. <a href="https://smcclatchy.github.io/qtl-mapping/index.html">Workshop overview</a></td>
        <td>What will we learn in this workshop?</td>
      </tr>      
      <tr><td>9:15</td>
        <td>1. <a href="https://smcclatchy.github.io/qtl-mapping/intro_data_set.html">Introduction to the Data Set</a></td>
        <td>What data will we be using in this workshop?</td>
      </tr>
      <tr><td>9:50</td>
        <td>2. <a href="https://smcclatchy.github.io/qtl-mapping/input-file-format.html">Input File Format</a></td>
        <td>How are the data files formatted for qtl2?
          <br>Which data files are required for qtl2?
          <br>Where can I find sample data for mapping with the qtl2 package?
        </td>
      </tr>
      <tr><td>10:35</td>
        <td>Break</td>
        <td></td>
      </tr>
      <tr>
        <td>10:50</td>
        <td>3. <a href="https://smcclatchy.github.io/qtl-mapping/calc-genoprob.html">Calculating Genotype Probabilities</a></td>
        <td>How do I calculate QTL at positions between genotyped markers?
          <br>How do I calculate QTL genotype probabilities?
          <br>How do I calculate allele probabilities?
          <br>How can I speed up calculations if I have a large data set?
        </td>
      </tr>
      <tr><td>12:00</td>
        <td>Lunch</td>
        <td></td>
      </tr>
      <tr>
        <td>1:00</td>
        <td>4. <a href="https://smcclatchy.github.io/qtl-mapping/perform-genome-scan.html">Performing a genome scan</a></td>
        <td>How do I perform a genome scan?<br>How do I plot a genome scan?
          <br>How do additive covariates differ from interactive covariates?
        </td>
      </tr>
      <tr><td>2:00</td>
        <td>5. <a href="https://smcclatchy.github.io/qtl-mapping/calc-kinship.html">Calculating A Kinship Matrix</a></td>
        <td>Why would I calculate kinship between individuals?
          <br>How do I calculate kinship between individuals?<br>What does a kinship matrix look like?
        </td>
      </tr>
      <tr><td>3:00</td>
        <td>Break</td>
        <td></td>
      </tr>
      <tr><td>3:15</td>
        <td>6. <a href="https://smcclatchy.github.io/qtl-mapping/perform-genome-scan-lmm.html">Performing a genome scan with a linear mixed model</a></td>
        <td>How do I use a linear mixed model in a genome scan?
          <br>How do different mapping and kinship calculation methods differ?
        </td>
      </tr>
      <tr><td>3:45</td>
        <td>7. <a href="https://smcclatchy.github.io/qtl-mapping/perform-genome-scan-bin.html">Performing a genome scan with binary traits</a></td>
        <td>How do I create a genome scan for binary traits?</td>
      </tr>
      <tr><td>5:00</td>
        <td>END</td>
        <td></td>
      </tr>
    </tbody>
  </table>
  <h3>Thursday, Nov 14</h3>
  <table class="table table schedule table-striped" role="presentation">
    <tbody>
      <tr><td>9:00</td>
        <td><a href="https://smcclatchy.github.io/qtl-mapping/index.html">Recap of day 1</a></td>
        <td>What did we learn yesterday and how will that knowledge be used today?</td>
      </tr>
      <tr><td>9:20</td>
        <td>8. <a href="https://smcclatchy.github.io/qtl-mapping/perform-perm-test.html">Performing a permutation test</a></td>
        <td>How can I evaluate the statistical significance of genome scan results?</td>
      </tr>
      <tr><td>9:50</td>
        <td>9. <a href="https://smcclatchy.github.io/qtl-mapping/find-lod-peaks.html">Finding QTL peaks</a></td>
        <td>How do I locate QTL peaks above a certain LOD threshold value?</td>
      </tr>
      <tr><td>10:35</td>
         <td>Break</td>
         <td></td>
      </tr>
      <tr>
        <td>10:50</td>
        <td>9. <a href="https://smcclatchy.github.io/qtl-mapping/find-lod-peaks.html">Finding QTL peaks</a></td>
        <td>How do I locate QTL peaks above a certain LOD threshold value?</td>
      </tr>
      <tr><td>12:00</td>
        <td>Lunch</td>
        <td></td>
      </tr>
      <tr>
        <td>1:00</td>
        <td>10. <a href="https://smcclatchy.github.io/qtl-mapping/est-qtl-effects.html">Estimating QTL effects</a></td>
        <td>How do I find the founder allele effects at a QTL peak?</td>
      </tr>
      <tr><td>1:30</td>
        <td>11. <a href="https://smcclatchy.github.io/qtl-mapping/integrating-gene-expression.html">Integrating Gene Expression Data</a></td>
        <td>How can I use gene expression data to identify candidate genes?
          <br>What is expression QTL mapping?</td>
      </tr>
      <tr><td>3:00</td>
        <td>Break</td>
        <td></td>
      </tr>
      <tr><td>3:15</td>
        <td>12. <a href="https://smcclatchy.github.io/qtl-mapping/do_qtl_mapping.html">QTL Mapping in Diversity Outbred Mice</a></td>
        <td>How do I map traits in Diversity Outbred mice?
          <br>How do I interpret the founder allele effects at a QTL peak?
          <br>How do I perform association mapping in Diversity Outbred mice?
          <br>How do I narrow down the set of candidate genes under a QTL?
        </td>
      </tr>
      <tr><td>5:00</td>
        <td>END</td>
        <td></td>
      </tr>
    </tbody>
  </table>
  <p>
    The actual schedule may vary slightly depending on the topics and exercises chosen by the instructor.
  </p>
