{% include install_instructions/r.html %}

<p>
After installing R and RStudio, start RStudio.
</p>
    
<h3>R packages</h3>

The [qtl2](https://github.com/rqtl/qtl2) package contains code for haplotype reconstruction, QTL mapping and plotting.   
The [qtl2convert](https://github.com/rqtl/qtl2convert) package contains code for converting data objects from one format to another.  
Install the following packages by copying and pasting the following code in the R console.  

~~~
install.packages(c("tidyverse", "ggbeeswarm", "qtl2", "qtl2convert"))
~~~
{: .r}

Once the installation is complete, load the libraries to make sure that they installed correctly.  

~~~
library(tidyverse)
library(ggbeeswarm)
library(qtl2)
library(qtl2convert)
~~~
{: .r}

If the libraries don't load and you received errors during the installation, please contact the workshop instructors before the workshop to help you.  

<h3>Project organization</h3>

1. Create a new project in your Desktop called `qtl_mapping`.  
- Click the `File` menu button, then `New Project`.  
- Click `New Directory`.  
- Click `New Project`.  
- Type `qtl_mapping` as the directory name. Browse to your Desktop to create the project there.  
- Click the `Create Project` button.  

2. Use the `Files` tab to create  a `data` folder to hold the data, a `scripts` folder to house your scripts, and a `results` folder to hold results. Alternatively, you can use the  R console to run the following commands for step 2 only. You still need to create a  project with step 1.  

~~~
dir.create("./data")
dir.create("./scripts")
dir.create("./results")
~~~
{: .r}

<h3>Data Sets</h3>

For this course, we will have several data files which you will need to download to the `data` directory in the project folder on your Desktop.
Copy, paste, and run the following code in the RStudio console.  

The first file contains the data that we will use for QTL mapping in an F2 population.  

~~~
download.file(url      = "https://thejacksonlaboratory.box.com/shared/static/svw7ivp5hhmd7vb8fy26tc53h7r85wez.zip",
              destfile = "data/attie_b6btbr_grcm39.Rdata",
              mode     = "wb")
~~~
{: .r}

The second file contains the Diversity Outbred mapping data.  

~~~
download.file(url      = "https://thejacksonlaboratory.box.com/shared/static/wspizp2jgrtngvvw5ixredpu7627mh5w.rdata",
              destfile = "data/qtl2_demo_grcm39.Rdata",
              mode     = "wb")
~~~
{: .r}

Next, download the MUGA marker positions from Karl Broman's Github page.  

~~~
download.file(url      = "https://raw.githubusercontent.com/kbroman/MUGAarrays/main/UWisc/muga_uwisc_v4.csv",
              destfile = "data/muga_uwisc_v4.csv",
              mode     = "wb")
~~~
{: .r}

Next, we need a database of the DO founder SNPs and gene positions. This file is 10 GB, so it will take a while to download.  

~~~
download.file(url      = "https://figshare.com/ndownloader/files/40157572",
              destfile = "data/fv.2021.snps.db3",
              mode     = "wb")
~~~
{: .r}

3. Please download the following large files **before the workshop**, and place them in your `data` folder. You can download the files from the URLs below and move the files the same way that you would for downloading and moving any other kind of data.  

- [cc_variants.sqlite doi:10.6084/m9.figshare.5280229.v2](https://figshare.com/articles/SQLite_database_of_variants_in_Collaborative_Cross_founder_mouse_strains/5280229/2), variants in the Collaborative Cross founders (3 GB)  
- [mouse_genes.sqlite doi:10.6084/m9.figshare.5280238.v4](https://figshare.com/articles/SQLite_database_with_mouse_gene_annotations_from_Mouse_Genome_Informatics_MGI_at_The_Jackson_Laboratory/5280238/4) full set of mouse gene annotations (677 MB)  
- [mouse_genes_mgi.sqlite doi:10.6084/m9.figshare.5286019.v5](https://figshare.com/articles/SQLite_database_with_MGI_mouse_gene_annotations_from_Mouse_Genome_Informatics_MGI_at_The_Jackson_Laboratory/5286019/5) just the MGI mouse gene annotations (11 MB)  

Alternatively, you can copy and paste the following into the R console to download the data.  

~~~
setwd("~/Desktop/mapping")
download.file("https://ndownloader.figshare.com/files/9746485", "./data/cc_variants.sqlite")
download.file("https://ndownloader.figshare.com/files/9746458", "./data/mouse_genes.sqlite")
download.file("https://ndownloader.figshare.com/files/9746452", "./data/mouse_genes_mgi.sqlite")
~~~
{: .r}
